app/Resources/views/pagelayout.html.twig

```diff
@@ -21,6 +22,12 @@
             <link rel="stylesheet" type="text/css" href="{{ asset_url }}">
         {% endstylesheets %}
 
+        {% javascripts
+            'assets/js/prism.js'
```

Comment:
I would prefer to include this file only on the blog post full view and not for every page of the website.
Possible?